build/yamls/antrea-aks.yml

```diff
@@ -2975,6 +2975,12 @@ data:
       # Note that setting ProxyLoadBalancerIPs to false usually only makes sense when ProxyAll is set to true and
       # kube-proxy is removed from the cluser, otherwise kube-proxy will still load-balance this traffic.
       #proxyLoadBalancerIPs: true
+
+    # The role in which Antrea Agent is working. Supported values:
```

Comment:
Again, can we generate a separate yaml for VM/BM, to avoid these confusing parameters for normal K8s?

Reply:
I create a sperate build directory for VM/BM configuration, but I didn't modify the create generate-manifest script to generate yaml file because for VM/BM case antrea-agent.conf is used directly on agent side and we don't need a all-in-one yaml file to apply on K8s cluster. Besides, In @mengdie-song 's PR, she will create a separate CRD yaml file for VM agent which could be applied on the cluster, and I think that file should also place in the new configration directory.

Reply:
I just mean could we find a way to avoid changing the regular yamls and adding VM only contents.